catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -13,6 +13,10 @@
  */
 package ddf.catalog.source.solr;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
I thought we were in the process of changing everything over to gson instead of boon and jackson

Reply:
Solr dependencies already require and include Jackson.  I did not like the idea of adding another JSON library to this project.